topology.ini

```diff
@@ -17,9 +17,3 @@ edge
 
 [postgresql:children]
 master3
-
-[postgresql_client:children]
```

Comment:
https://github.com/TOSIT-IO/tdp-collection-prerequisites/blob/272ae7d3bb7e194d94b2d3607f8c1ddbb51de516/playbooks/postgresql.yml#L10
Above playbook use `postgresql_client`, if you remove it there is no warning ? If there are warnings, I think you should keep an empty `postgresql_client` group.

Reply:
Indeed, it crashes if postgresql_client section is not defined in any topology. We need to keep an empty section